src/ocm/components/clusterConfiguration/review/ReviewOperatorsTable.tsx

```diff
@@ -16,42 +13,18 @@ export const ReviewOperatorsTable = ({ cluster }: { cluster: Cluster }) => {
   const operatorNames = operatorLabels(t);
 
   const rows = React.useMemo(() => {
-    return [
-      {
-        cells: [
-          operatorNames[OPERATOR_NAME_CNV],
-          {
-            title: hasEnabledOperators(cluster.monitoredOperators, OPERATOR_NAME_CNV)
-              ? 'Enabled'
-              : 'Disabled',
-            props: { 'data-testid': 'openshift-virtualization' },
-          },
-        ],
-      },
-      !isSNO(cluster) && {
-        cells: [
-          operatorNames[OPERATOR_NAME_ODF],
-          {
-            title: hasEnabledOperators(cluster.monitoredOperators, OPERATOR_NAME_ODF)
-              ? 'Enabled'
-              : 'Disabled',
-            props: { 'data-testid': 'openshift-data-foundation' },
-          },
-        ],
-      },
-      isSNO(cluster) && {
-        cells: [
-          operatorNames[OPERATOR_NAME_LVM],
-          {
-            title: hasEnabledOperators(cluster.monitoredOperators, OPERATOR_NAME_LVM)
-              ? 'Enabled'
-              : 'Disabled',
-            props: { 'data-testid': 'openshift-data-foundation' },
-          },
-        ],
-      },
-    ];
-  }, [cluster, operatorNames]);
+    return ExposedOperatorNames.filter((operator) =>
+      hasEnabledOperators(cluster.monitoredOperators, operator),
```

Comment:
You have fixed some Eslint warnings by properly typing the data, please update `package.json`